packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/tera-calibrate-ensemble-ciemss-drilldown.vue

```diff
@@ -190,50 +190,75 @@
 		</section>
 		<template #preview>
 			<tera-drilldown-section>
-				<!-- Loss chart -->
-				<div ref="lossChartContainer">
-					<vega-chart
-						v-if="!_.isEmpty(lossValues)"
-						expandable
-						ref="lossChartRef"
-						:are-embed-actions-visible="true"
-						:visualization-spec="lossChartSpec"
-					/>
-				</div>
-				<section v-if="!inProgressCalibrationId && !inProgressForecastId" ref="outputPanel">
-					<tera-simulate-chart
-						v-for="(cfg, index) of node.state.chartConfigs"
-						:key="index"
-						:run-results="runResults"
-						:chartConfig="{
-							selectedRun: props.node.state.postForecastId,
-							selectedVariable: cfg
-						}"
-						has-mean-line
-						@configuration-change="chartProxy.configurationChange(index, $event)"
-						@remove="chartProxy.removeChart(index)"
-						show-remove-button
-						:size="chartSize"
-						class="mb-2"
-					/>
-					<Button
-						class="p-button-sm p-button-text"
-						@click="chartProxy.addChart()"
-						label="Add chart"
-						icon="pi pi-plus"
-					/>
+				<section class="pb-3" ref="outputPanel">
+					<Accordion multiple :active-index="[0, 1]" class="px-2">
```

Comment:
We will want to store the active index in a ref `currentActiveIndexes`

Reply:
Let's address this in a separate issue.